rate_limiter/src/lib.rs

```diff
@@ -410,13 +410,12 @@ impl RateLimiter {
             ops_complete_refill_time_ms,
         );
 
-        // If limiting is disabled on all token types, don't even create a timer fd.
-        let timer_fd = if bytes_token_bucket.is_some() || ops_token_bucket.is_some() {
-            // create TimerFd using monotonic clock, as nonblocking FD and set close-on-exec
-            Some(TimerFd::new_custom(ClockId::Monotonic, true, true)?)
-        } else {
-            None
-        };
+        // TODO: Self::timer_fd should not be an `Option` anymore; clean that up.
```

Comment:
Can you do this as part of this PR? I don't think it will get done **later**.

Reply:
Done.